Dockerfile.aaq-controller

```diff
@@ -1,33 +1,37 @@
 # Builder stage
-FROM docker.io/library/golang:1.22-alpine as builder
+FROM  --platform=${BUILDPLATFORM} docker.io/library/golang:1.23-alpine as builder
+ARG BUILDPLATFORM
 
 # Install make
 RUN apk update && apk add make
 
-# Copy the source code from the host to the container
-COPY pkg /workdir/app/pkg
-COPY staging /workdir/app/staging
-COPY cmd /workdir/app/cmd
-COPY vendor /workdir/app/vendor
-COPY go.mod /workdir/app/go.mod
-COPY go.sum /workdir/app/go.sum
-COPY Makefile /workdir/app/Makefile
-
 WORKDIR /workdir/app
 
-RUN make aaq_controller
+# Copy the source code from the host to the container
+COPY pkg ./pkg
+COPY staging ./staging
+COPY cmd ./cmd
+COPY vendor ./vendor
+COPY go.mod ./go.mod
+COPY go.sum ./go.sum
+COPY Makefile ./Makefile
+
+ARG TARGETOS
```

Comment:
I don't think we build the images for windows, so this one is not needed, or could just be linux by default.

Reply:
Yes it will be set by podman/docker, but we have no reason to use it.